Index.html

```diff
@@ -0,0 +1,31 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio-Setup-and mobile-first</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <header>
+    <div class="hamburguer">☰</div>
+  </header>
+  <main>
+    <section class="hello">
+        <img class="image1"src="./images/App Bar - Compact - Dark - iOS.png">
+        <img class="socialmedia"src="./images/Social Media.png">
```

Comment:
Please keep in mind that you need to add your own icons with your links. This will improve the user experience and allows anyone to contact/check your social profiles with ease 😄. You should do it by creating an [ul](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) and each `li` inside it should contain an `a` tag linking to your profile.

Here is my tip for you:

```html
<ul>
  <li><a href="your social media URL">the social media icon</a></li>
</ul
```